codegen-server-test/model/rest_json_extras.smithy

```diff
@@ -0,0 +1,126 @@
+$version: "1.0"
```

Comment:
Don't copy the contents of this file. Symlink it so that both client and server use the same model. See `simple.smithy` for example.

```
➜ file codegen-test/model/simple.smithy
codegen-test/model/simple.smithy: symbolic link to ../../codegen-server-test/model/simple.smithy
```